daq_macros.py

```diff
@@ -1932,8 +1935,7 @@ def snakeRasterBluesky(rasterReqID, grain=""):
       db_lib.updateRequest(rasterRequest)
       daq_lib.set_field("xrecRasterFlag",rasterRequest["uid"])
       logger.info(f'setting xrecRasterFlag to: {rasterRequest["uid"]}')
-    raster_flyer.detector.cam.acquire.put(0)
-    raster_flyer.detector.super_unstage()
+    raster_flyer.detector.unstage()
```

Comment:
What does this function do?

Reply:
Or I guess that's clear, but why are we changing it in for acquire.put(0)?

Reply:
in this pull request, I removed hard-coding detector acquire stop into the main body of the collection code, which is replaced by the finalize function. `super_unstage()` was a previous attempt to just do `unstage()` of the detector instead of stopping the detector.

`unstage()` always needs to be done at the end of a collection after hardware is staged (in the case of this Eiger, setting up the prefix and data directory). the `stage()` function will do things like make sure the detector isn't staged already and get it ready to go. we are not using this to arm the detector, btw, because we want to use the time while arming to do other things (like governor changes). (though we are currently out of order with arming and governor changes.)